script/workflows/generate-release-notes.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# this script is used to generate release notes for a given release
+# first argument is the pull request id for the last release
+# the second is the new release number
+
+# the script then grabs every pull request merged since that pull request
+# and outputs a string of release notes
+
+echo "Generating release notes for $2"
```

Comment:
another nit (feel free to ignore): You are defining a variable below `LAST_RELEASE` but not using it above. `$2` is sort of a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)).

Reply:
Fixed!